numpy/core/tests/test_einsum.py

```diff
@@ -605,6 +605,11 @@ def test_einsum_misc(self):
                      [[[1,  3], [3,  9], [5, 15], [7, 21]],
                      [[8, 16], [16, 32], [24, 48], [32, 64]]])
 
+        # Ensure explicitly setting out=None does not cause an error
+        # see issue #15776 and issue #15256
+        assert_equal(np.einsum('i,j', [1], [2], out=None),
+                               [[2]])
```

Comment:
nit: Indents are misleading here:
```suggestion
        assert_equal(np.einsum('i,j', [1], [2], out=None), [[2]])
```

Reply:
Ah OK! I though from the context that test results should go on a new line. But sure, from the line length, it can be on one line.

Reply:
If you wanted to do that, the wrapping would be
```suggestion
        assert_equal(np.einsum('i,j', [1], [2], out=None),
                     [[2]])
```
The way you wrapped it made it look like an argument to einsum.

But in general, test results on the same line are fine for short tests.